chrome/content/preferences/prefs.js

```diff
@@ -474,7 +474,7 @@ var Servers = {
 		while (this._list.firstChild){
 			this._list.removeChild(this._list.firstChild);
 		}
-		for (let [,limit] in Iterator(this.listLimits())) {
+		for (let [,limit] of Object.entries(this.listLimits())) {
```

Comment:
In the upstream master branch, Nils already updated this:
```suggestion
		for (let [,limit] of this.listLimits()) {
```
Apparently the line `requireJoined(Servers, "support/serverlimits");` (see modules/glue.jsm:497) pulls in the definition of `listLimits()` method of `Server`, defined as the `.entries()` of a map object defined there. So the upstream version looks like it's OK. What do you think?